### PR TITLE
Added controller_type param needed by imcsdk

### PR DESCRIPTION
**Pull request by dsoper2**
Added boot_drive support also

When adding the boot drive support, I also noticed that when checking for an existing virtual drive, mo properties are not compared.  If a user does set boot_drive (or other virtual drive params) in the web UI for example, the module won't currently catch the state change.

### CODE REVIEW

File: library/cisco_imc_virtual_drive.py [hunk 1]
```diff
@@ -186,8 +198,10 @@ def virtual_drive(server, module):
                 results["changed"] = not _exists
                 return results, False
 
+            print "size is %s" % ansible["size"]
```
Comment (vvb) [inline library/cisco_imc_virtual_drive.py:201]: @dsoper2 ansible frowns upon any print statements. we should probably remove this line.

**Review by vvb — APPROVED** [review]
LGTM.

File: library/cisco_imc_virtual_drive.py (hunk 1 above)
Comment (vvb) [reply library/cisco_imc_virtual_drive.py:201]: I am merging this right now and will take care of this comment.

**Comment by dsoper2** [comment]
My fault on that - did not clear out my debug before committing!

On Feb 7, 2017, at 6:39 PM, Vikrant Balyan <notifications@github.com<mailto:notifications@github.com>> wrote:


@vvb commented on this pull request.

________________________________

In library/cisco_imc_virtual_drive.py<https://github.com/CiscoUcs/imc-ansible/pull/2#pullrequestreview-20658918>:

> @@ -186,8 +198,10 @@ def virtual_drive(server, module):
                 results["changed"] = not _exists
                 return results, False

+            print "size is %s" % ansible["size"]


@dsoper2<https://github.com/dsoper2> ansible frowns upon any print statements. we should probably remove this line.

-

**Pull request merged**